src/main/java/com/drullkus/thermalsmeltery/ThermalSmeltery.java

```diff
@@ -22,6 +22,9 @@
 @Mod(modid = LibMisc.MOD_ID, name = LibMisc.MOD_NAME, dependencies = LibMisc.DEPENDENCIES)
 public class ThermalSmeltery
 {
+    @Mod.Instance(value = LibMisc.MOD_ID)
+    public static ThermalSmeltery instance = new ThermalSmeltery();
```

Comment:
It doesn't make a difference, FML loads it if it hasn't already been loaded - I personally prefer calling it myself